TaskHero/TaskCatalogViewController.swift

```diff
@@ -21,42 +21,48 @@ class TaskCatalogViewController: UIViewController {
 
         tableView.dataSource = self
         tableView.delegate = self
-        
         tableView.rowHeight = UITableViewAutomaticDimension
         tableView.estimatedRowHeight = 160
-        
         tableView.register(UINib(nibName: taskCardCellIdentifier, bundle: nil), forCellReuseIdentifier: taskCardCellIdentifier)
         
-        // Use temporary data for now
-        tasks = DummyTaskData.getTaskData()
-        tableView.reloadData()
+        loadTasks()
     }
 
+    func loadTasks() {
+        ParseClient.sharedInstance.getAllTasks(sucess: {(tasks) -> () in
+            self.tasks = tasks
+            self.tableView.reloadData()
+        }, failure: {(error) -> () in
+            NSLog("Error: \(error)")
+        })
+    }
+    
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
     }
     
     // MARK: - Navigation
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        if segue.identifier == "TaskCatalogToTaskView" {
+        if segue.identifier == "TaskCatalogToTaslCatalogDetail" {
```

Comment:
spelling :) "TaskCatalogToTaslCatalogDetail", probably should be "TaskCatalogToTaskCatalogDetail"